mix.exs

```diff
@@ -30,8 +30,7 @@ defmodule Evision.MixProject do
           System.get_env("MAKE_BUILD_FLAGS", "-j#{System.schedulers_online()}"),
         "CMAKE_OPTIONS" => cmake_options,
         "ENABLED_CV_MODULES" => enabled_modules
-      },
-      xref: [exclude: [Nx]]
```

Comment:
`xref: [exclude: [Nx]]` will be deprecated in future releases. `@compile {:no_warn_undefined, Nx}` in the module calling Nx should be enough!